index.html

```diff
@@ -200,5 +200,64 @@ <h4>Skills</h4>
       </ul>
     </div>
   </section>
+  <section id="contact-form" class="flex-column">
+    <p>I'm always <br>interested in hearing about new projects, 
+      so if you'd like to chat please get in touch.</p>
+    <form action="https://formspree.io/f/xzbykgyg" method="POST">
+      <ul class="flex-column">
+        <li id="name-label">
+          <label>
+            <input type="text" id="name" name="name" placeholder="Full name" maxlength="30" 
+            required>
+          </label>
+        </li>
+        <li>
+          <hr>
+          <label>
+            <input type="email" id="email" name="email" placeholder="Email address" required>
+          </label>
+        </li>
+        <li>
+          <hr>
+          <textarea name="message" id="message" cols="31" rows="5" 
+          placeholder="Enter text here" maxlength="500" required></textarea>
+        </li>
+        <li>
+          <button id="form-button" type="submit">Get in touch</button>
+        </li>
+      </ul>
```

Comment:
- [ ] There are some issues with your inputs, please see the screenshots below 👇 
![image](https://user-images.githubusercontent.com/55465133/129810218-f3872db1-1ece-4ce5-8d60-e7f5f02298e0.png)

Your inputs are not taking the full width they were supposed to take, even the `<textarea>`.  see the difference with the one in Figma. 
![image](https://user-images.githubusercontent.com/55465133/129810460-9db3312d-77e3-48af-9caf-208602f77f90.png)
 
Can we please fix this issue?